README.md

```diff
@@ -0,0 +1,29 @@
+# Portfolio mobile version
+***
+>Microverse portfolio mobile template. This is third in five portfolio tamplates provided in figma.
+It is just the main page of the website and other pages will be updated in upcoming projects.
+
+## Built with
+* HTML
+* CSS
+* Microverse [linters](https://github.com/microverseinc/linters-config/tree/master/html-css)
+* [Figma](figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A988)
+***
+## Getting started
```

Comment:
**OPTIONAL** - To make your repo look more professional, I'd suggest you include steps on how to clone, navigate and open your project locally. Please fix this 👍 👍